### PR TITLE
Added node ready latency metric and used corrected formatting

### PR DESCRIPTION
**Pull request by vishnuchalla**
### Description
This PR adds a new metric, to understand duration for OVN node to get to a ready state.

### Metric Screenshots

Screenshot from 2023-02-06 21-44-47

Screenshot from 2023-02-06 21-44-53

Screenshot from 2023-02-06 21-44-57

### CODE REVIEW

**Review by krishvoor — APPROVED** [review]
lgtm

File: templates/ovn-dashboard.jsonnet [hunk 1]
```diff
@@ -132,6 +132,13 @@ local sync_latency = genericGraphLegendPanel('Sync Service Latency', 's').addTar
   )
 );
 
+local ovnkube_node_ready_latency = genericGraphLegendPanel('OVNKube Node Ready Latency', 's').addTarget(
+  prometheus.target(
+    'ovnkube_node_ready_duration_seconds{pod=~"ovnkube-node-.*",namespace="openshift-ovn-kubernetes",container!~"POD|"}',
+    legendFormat='{{pod}} - Time taken to get to ready state',
```
Comment (dry923) [inline templates/ovn-dashboard.jsonnet:138]: lets remove the "Time taken..." part from the legend. It is already in the title of the panel.

Comment (vishnuchalla) [reply templates/ovn-dashboard.jsonnet]: Updated as suggested.

**Review by rsevilla87 — APPROVED** [review]
lgtm

**Pull request merged**